glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: github.com/giantswarm/azure-operator
 import:
 - package: github.com/giantswarm/azuretpr
-  version: 596d9524fd8ec872442a8d32e42baf76438d3031
+  version: master
```

Comment:
I did it only for that and this wasn't updated ....

Reply:
Looks like in previous rebase vendor was changed but no glide rerun.
```
~/go/src/github.com/giantswarm/azure-operator/vendor/github.com/giantswarm/azuretpr (vendor2) (kube:giantswarm-hbsb1/default@)
$ ls spec/azure/virtualnetwork/loadbalancer.go
ls: spec/azure/virtualnetwork/loadbalancer.go: No such file or directory

~/go/src/github.com/giantswarm/azure-operator/vendor/github.com/giantswarm/azuretpr (vendor2) (kube:giantswarm-hbsb1/default@)
$ ls spec/azure/virtualnetwork/
ls: spec/azure/virtualnetwork/: No such file or directory

~/go/src/github.com/giantswarm/azure-operator/vendor/github.com/giantswarm/azuretpr (vendor2) (kube:giantswarm-hbsb1/default@)
$ ls spec/azure/
dns_zones.go       keyvault.go        node               node.go            storage.go         virtual_network.go

~/go/src/github.com/giantswarm/azure-operator/vendor/github.com/giantswarm/azuretpr (vendor2) (kube:giantswarm-hbsb1/default@)
```

Reply:
Yes glide.lock wasn't in sync with glide.yaml: https://github.com/giantswarm/azure-operator/pull/53/files#diff-f16a80eae23d5b298c2652448ec420cfR49